test/functional/pdo_sqlsrv/pdo_1391_string_truncation_after_short_string.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+GitHub issue 1391 - string truncation error when binding some parameters as longer strings the second time
+--DESCRIPTION--
+The test shows the same parameters, though bound as short strings in the first insertion, can be bound as longer strings in the subsequent insertions.
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--FILE--
+<?php
+
+require_once("MsSetup.inc");
+
+function dropTable($conn, $tableName)
+{
+    $drop = "IF EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'" . $tableName . "') AND type in (N'U')) DROP TABLE $tableName";
+    $conn->exec($drop);
+}
+
+try {
+    $conn = new PDO("sqlsrv:server=$server; Database = $databaseName;", $uid, $pwd);
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    dropTable($conn, 'domains');

```

Comment:
Should the table name be `long_strings`? Also, could you change your PR to `dev` branch, we don't typically merge to `master` until the next release. 
```suggestion
    dropTable($conn, 'long_strings');

```